dashboards-observability/public/components/metrics/helpers/utils.tsx

```diff
@@ -2,15 +2,19 @@
  * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
  */
+/* eslint-disable no-console */
```

Comment:
why was this added here?

Reply:
It prevents the error in `console.error()` as shown in [this file](https://github.com/opensearch-project/observability/blob/edbc7cb915a26fbbf590cb2f7e1445a50233a0e8/dashboards-observability/public/components/custom_panels/helpers/utils.tsx#L5) but if we remove the function all together, I can get rid of it.